yt/admin/dashboards/yt_dashboards/bin/main.py

```diff
@@ -54,10 +65,21 @@
             "args": [False],
         },
     },
+    # Supported backends: monitoring, grafana.
```

Comment:
Current convention is that Grafana backend is supported iff there is `"grafana"` section in the map, so the comment looks unnecessary.

Reply:
Reasonable.